src/aws/osml/model_runner/inference/feature_selection.py

```diff
@@ -68,9 +68,10 @@ def _get_lists_from_features(self, feature_list: List[Feature]) -> Tuple[List, L
         self.extents = [None, None, None, None]  # [min_x, min_y, max_x, max_y]
         self.feature_id_map = OrderedDict()
         self.labels_map = dict()
+
         for feature in feature_list:
+            bounds_imcoords = get_feature_image_bounds(feature)
             # imcoords: [x1, y1, x2, y2]
```

Comment:
Misplaced comment? Should be above line 73

Reply:
Might be out of scope, maybe this is on me for not reviewing the prior PRs. I noticed that some of the function docstring is incomplete.